### PR TITLE
pkgs: add brutal vscode wrapper

### PR DESCRIPTION
**Pull request by steveeJ**

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -36,12 +36,6 @@ let
           newDependency = overrides.libxcb_x2go;
         };
 
-        labshell = callPackage ./pkgs/labshell { };
```
Comment (mmaechtel) [inline default.nix]: Why is this deleted?

File: pkgs/default.nix [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    phases = "installPhase";
+    installPhase = ''
+      set -xe
+      mkdir -p $out/bin
+      cp -a ${src} $out/bin/code_pkill
+    '';
+  };
+
+  customLesspipe = mkDerivation {
```
Comment (mmaechtel) [inline pkgs/default.nix]: what is lesspipe?

File: default.nix (hunk 1 above)
Comment (steveeJ) [reply default.nix]: I have moved derivations to *pkgs/default.nix*, look further down to see it's inherited instead

File: pkgs/default.nix (hunk 2 above)
Comment (steveeJ) [reply pkgs/default.nix]: something that causes trouble on travis. I'm not sure if it's required anymore but it was at one point to make travis work

**Comment by steveeJ** [comment]
```
------------------------------------------------------------------
Tests completed successfully.
------------------------------------------------------------------
```
